centreon/www/include/configuration/configObject/host/DB-Func.php

```
@@ -111,10 +111,9 @@ function hostMacHandler()
     }
 
     $fieldsToBind = [];
-    $counter = count($_POST['macroInput']);
-    for ($index = 0; $index < $counter; $index++) {
-        $fieldsToBind[':macro_' . $index]
-            = "'\$_HOST" . strtoupper($_POST['macroInput'][$index]) . "\$'";
+    foreach ($_POST['macroInput'] as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)